Add Istio docs project

craigbox opened the pull request:
(Sorry for cutting it close to the wire; I've been on dad duty while mum has been sick the past week.)

Add Istio docs project, and make copy edits to Istio multi-cluster project.

nate-double-u commented:
We can get this in, but the PR is very confusing. Could you resolve the conflicts before I take a look at this?

craigbox commented:
Updated the PR to add to the readme, not the ideas doc.
I removed the copy edits I made to the other Istio project as you've probably already set them up on the LFX platform.  Let me know if it's worth sending another PR.

nate-double-u reviewed (approved):
Thanks for this @craigbox!

Two updates based on creating the LFX project.

Hold for LFX approval.

nate-double-u commented:
LFX approved
